config/source/multi/deployments/adapter.yaml

```diff
@@ -57,7 +57,7 @@ spec:
         # DO NOT MODIFY: The values below are being filled by the kafka source controller
         # See 500-controller.yaml
         - name: K_METRICS_CONFIG
-          value: '{"Domain":"knative.dev/eventing","Component":"kafkasource","ConfigMap":{"profiling.enable": "true"}}'
+          value: '{"Domain":"knative.dev/eventing","Component":"kafkasource","ConfigMap":{"profiling.enable": "false"}}'
```

Comment:
what is profiling.enable for? Is it for sarama logging?

Reply:
it's for enabling Golang profiling. Shouldn't be enabled by default.